explorer/src/tests.rs

```diff
@@ -0,0 +1,76 @@
+use chain_core::property::FromStr;
+
+use jormungandr_automation::jormungandr::JormungandrProcess;
+use jormungandr_automation::jormungandr::{
+    explorer::configuration::ExplorerParams, ConfigurationBuilder, Explorer,
+};
+
+use jormungandr_integration_tests::startup;
+use jormungandr_lib::crypto::hash::Hash;
+use jormungandr_lib::interfaces::ActiveSlotCoefficient;
+
+pub struct ExplorerTestConfig {
```

Comment:
My guess is that as this struct grows, we probably don't want to have to always specify the values. A `Default` implementation is probably handy so we can just override the values we care about for a specific test.

Not sure what a sensible default is, but I'm sure there's some other tests to copy from.